src/main/java/ch/njol/skript/entity/BoatChestData.java

```diff
@@ -32,21 +33,27 @@
 
 public class BoatChestData extends EntityData<ChestBoat> {
 
-	private static ItemType oakBoat = null;
-	private static ItemType spruceBoat = null;
-	private static ItemType birchBoat = null;
-	private static ItemType jungleBoat = null;
-	private static ItemType acaciaBoat = null;
-	private static ItemType darkOakBoat = null;
+	private static Material oakBoat = null;
+	private static Material spruceBoat = null;
+	private static Material birchBoat = null;
+	private static Material jungleBoat = null;
+	private static Material acaciaBoat = null;
+	private static Material darkOakBoat = null;
 
 	static {
 		if (Skript.classExists("org.bukkit.entity.ChestBoat")) {
-			oakBoat = Aliases.javaItemType("oak chest boat");
-			spruceBoat = Aliases.javaItemType("spruce chest boat");
-			birchBoat = Aliases.javaItemType("birch chest boat");
-			jungleBoat = Aliases.javaItemType("jungle chest boat");
-			acaciaBoat = Aliases.javaItemType("acacia chest boat");
-			darkOakBoat = Aliases.javaItemType("dark oak chest boat");
+			//noinspection ConstantConditions
+			oakBoat = Material.getMaterial("OAK_CHEST_BOAT");
```

Comment:
it should be safe to reference the enum values directly since registration is locked behind this check